tests/testthat/test_array_manipulation.r

```diff
@@ -46,9 +46,17 @@ test_that('get.limit.index sets the right indexes', {
   expect_equal(lim_index$end, c(7, 8, 9, 10))  
 })
 
+
 test_that('ellip.limit.vec returns correctly for numerics', {
-  ellip.h <- '\u22EF'
-  
+  # copy of the function repr_matrix_df.r
+  .char_fallback <- function(char, default) {
```

Comment:
as you can see in the link, that’s what travis uses to produce that output, as well. hmm.

anyway: the testthat package has a function to run the tests, you can try that one.